test/anonymize_proxy.js

```diff
@@ -1,37 +1,47 @@
-const _ = require('underscore');
-const util = require('util');
-const { expect, assert } = require('chai');
-const proxy = require('proxy');
-const http = require('http');
-const portastic = require('portastic');
-const basicAuthParser = require('basic-auth-parser');
-const request = require('request');
-const express = require('express');
-
-const { anonymizeProxy, closeAnonymizedProxy, listenConnectAnonymizedProxy } = require('../src/index');
+const _ = require("underscore");
```

Comment:
Can you revert those code style changes?

Reply:
@szmarczak done